open-company/open-company-web
# Better static debug

We have some utility functions we use to debug the data in the app-state. Until now these functions were printing the data to the console with `js/console.log` instead of returning them.
Returning the value means you can manipulate them, count them etc...
Also i added a new object to these static functions called `OCWebUtils` that is used to have a shortcut to some core functions of cljs that usually are not usable with advanced configuration: `deref`, `keyword`, `count`, `get`, `filter`, `map`, `clj->js` and `js->clj`.

To test:
- use chrome that has formatters (even if this should work on other consoles too, it's more readable on Chrome)
- deploy to staging (should be already there)
- [x] make sure site is working as usual
- now open the console
- insert this `var must_see_posts=OCWebUtils.filter(function(i){return OCWebUtils.get(OCWebUtils.get(i, 1), OCWebUtils.keyword("must-see"))}, OCWebPrintPostsData());`
- now check what's `must_see_posts` value, should be all the must see posts
- you can count the values in it (.length won't work): `OCWebUtils.count(must_see_posts)`
- merge
- deploy
